app/src/main/java/com/example/musicappmvp/ui/adapter/SongAdapter.kt

```diff
@@ -0,0 +1,54 @@
+package com.example.musicappmvp.ui.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.example.musicappmvp.R
+import com.example.musicappmvp.model.Song
+import kotlinx.android.synthetic.main.item_song.view.*
+
+class songAdapter(var clickItem: ((Int) -> Unit)) :
```

Comment:
sua lai ten class ne

Reply:
```suggestion
class SongAdapter(private val clickItem: ((Int) -> Unit)) :
```
ông truyền higher order function bên activity , để private thôi